Include/httpClient/httpClient.h

```diff
@@ -381,7 +381,7 @@ STDAPI HCHttpCallRequestSetRequestBodyString(
     ) noexcept;
 
 #if !HC_NOZLIB
-#if HC_PLATFORM == HC_PLATFORM_WIN32 || HC_PLATFORM == HC_PLATFORM_GDK || HC_PLATFORM == HC_PLATFORM_NINTENDO_SWITCH
+#if HC_PLATFORM == HC_PLATFORM_WIN32 || HC_PLATFORM == HC_PLATFORM_GDK || HC_PLATFORM == HC_PLATFORM_NINTENDO_SWITCH || HC_PLATFORM == HC_PLATFORM_IOS || HC_PLATFORM == HC_PLATFORM_MAC
```

Comment:
could you replace with: HC_PLATFORM_IS_APPLE? Same applies to other places.

Reply:
sure, that makes sense!

Reply:
Fixed